practice/spec/regex/regex_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rspec'
+require_relative '../../../practice/lib/regex/functionality/regex_patterns'
+
+describe 'Regex pattern' do
+  before(:each) do
+    @file_name = 'Story.txt'
+  end
+
+  it 'should not found not existing string' do
+    expect(RegexPatterns.zero_occurrence?('Story.txt', 'This can not be in book')).to eq true
```

Comment:
Sometimes it's difficult to understand large expect. We can do each step in a new line. For example:
```
matches = Module.get_number_of_matches
expect(matches).to eq 0
```